src/azure-cli-testsdk/azure/cli/testsdk/base.py

```diff
@@ -3,29 +3,29 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
-import unittest
 import os
-import inspect
 import subprocess
 import json
 import shlex
-import tempfile
-import shutil
 import logging
-import six
-import vcr
+import inspect
+import sys
+
+from azure_devtools.scenario_tests.base import LiveTest
+from azure_devtools.scenario_tests.base import ScenarioTest as DevToolsScenarioTest
+from azure_devtools.scenario_tests.recording_processors import (
+    SubscriptionRecordingProcessor, OAuthRequestResponsesFilter,
+    GeneralNameReplacer, LargeRequestBodyProcessor,
+    LargeResponseBodyProcessor, LargeResponseBodyReplacer,
+    DeploymentNameReplacer
+)
+from azure_devtools.scenario_tests.utilities import create_random_name
```

Comment:
No. `from xxx import *` is a bad practice and no one should do. I want to simply the namespace. I do not want the users to get components from different namespaces because it is not easy to figure out. All names should be exposed in the same package. https://github.com/Azure/azure-cli/blob/master/src/azure-cli-testsdk/azure/cli/testsdk/__init__.py